docs/index.bs

```diff
@@ -1861,7 +1863,12 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
                     1. Add a copy of |requestResponse|'s response to |responses|.
             1. Else:
                 1. Let |requestResponses| be the result of running [=Query Cache=] with |r| and |options|.
+                1. Let |request for CORP check| be a copy of |r|.
+                1. Set |request for CORP check|'s [=request/mode=] to "<code>no-cors</code>".
+                1. Set |request for CORP check|'s [=request/origin=] to |promise|'s [=relevant settings object=]'s [=environment settings object/origin=].
+                1. Set |request for CORP check|'s [=request/client=] to |promise|'s [=relevant settings object=].
                 1. [=list/For each=] |requestResponse| of |requestResponses|:
+                    1. If |requestResponse|'s [=response/type=] is "<code>opaque</code>" and [=cross-origin resource policy check=] with |request for CORP check| and |requestResponse| returns <b>blocked</b>, then reject |promise| with a `TypeError` and abort these steps.
```

Comment:
Can we provide a better abstraction for this caller whereby you give an origin and a response and get an answer?

It's also not clear to me that the current algorithm does the right thing here as it's not using the URL from the response to compare with the request origin, but rather the URL of the request.

Reply:
Done, I believe.